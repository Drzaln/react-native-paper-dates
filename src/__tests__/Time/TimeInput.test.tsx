import { render } from '@testing-library/react-native'
import TimeInput from '../../Time/TimeInput'
import { clockTypes } from '../../Time/timeUtils'

it('renders TimeInput', () => {
  const { toJSON } = render(
    <TimeInput
      value={12}
      clockType={clockTypes.hours}
      pressed
      inputType={'picker'}
      onChanged={() => null}
    />
  )

  expect(toJSON()).toMatchSnapshot()
})
