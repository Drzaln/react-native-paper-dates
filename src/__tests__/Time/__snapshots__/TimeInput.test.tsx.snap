// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TimeInput 1`] = `null`;
